misk-testing/src/main/kotlin/misk/testing/MiskTestExtension.kt

```diff
@@ -18,6 +19,7 @@ internal class MiskTestExtension : BeforeEachCallback, AfterEachCallback {
   override fun beforeEach(context: ExtensionContext) {
     val module = object : KAbstractModule() {
       override fun configure() {
+        Environment.setTesting()
```

Comment:
why do this in the `configure` method? makes more sense outside (especially since some modules depend on this env var unfortunately)